main.py

```diff
@@ -1 +1,18 @@
 #Mad Libs Generator Project
+verb_1 = input("Enter a verb of choice, and press enter:")
+adj_1 = input("Enter a adjective of choice, and press enter:")
```

Comment:
Look into using [strip() ](https://www.w3schools.com/python/ref_string_strip.asp)to provide space in user input